src/Workspaces/Core/Portable/Workspace/Solution/ChecksumCollection.cs

```diff
@@ -87,42 +87,42 @@ IEnumerator IEnumerable.GetEnumerator()
     internal class ProjectChecksumCollection : ChecksumCollection
     {
         public ProjectChecksumCollection(Checksum[] checksums) : this((object[])checksums) { }
-        public ProjectChecksumCollection(object[] checksums) : base(WellKnownSynchronizationKind.ProjectChecksumCollection, checksums) { }
+        public ProjectChecksumCollection(object[] checksums) : base(checksums) { }
     }
 
     internal class DocumentChecksumCollection : ChecksumCollection
     {
         public DocumentChecksumCollection(Checksum[] checksums) : this((object[])checksums) { }
-        public DocumentChecksumCollection(object[] checksums) : base(WellKnownSynchronizationKind.DocumentChecksumCollection, checksums) { }
+        public DocumentChecksumCollection(object[] checksums) : base(checksums) { }
     }
 
     internal class TextDocumentChecksumCollection : ChecksumCollection
     {
         public TextDocumentChecksumCollection(Checksum[] checksums) : this((object[])checksums) { }
-        public TextDocumentChecksumCollection(object[] checksums) : base(WellKnownSynchronizationKind.TextDocumentChecksumCollection, checksums) { }
+        public TextDocumentChecksumCollection(object[] checksums) : base(checksums) { }
     }
 
     internal class AnalyzerConfigDocumentChecksumCollection : ChecksumCollection
     {
         public AnalyzerConfigDocumentChecksumCollection(Checksum[] checksums) : this((object[])checksums) { }
-        public AnalyzerConfigDocumentChecksumCollection(object[] checksums) : base(WellKnownSynchronizationKind.AnalyzerConfigDocumentChecksumCollection, checksums) { }
+        public AnalyzerConfigDocumentChecksumCollection(object[] checksums) : base(checksums) { }
     }
 
     internal class ProjectReferenceChecksumCollection : ChecksumCollection
     {
         public ProjectReferenceChecksumCollection(Checksum[] checksums) : this((object[])checksums) { }
-        public ProjectReferenceChecksumCollection(object[] checksums) : base(WellKnownSynchronizationKind.ProjectReferenceChecksumCollection, checksums) { }
+        public ProjectReferenceChecksumCollection(object[] checksums) : base(checksums) { }
     }
 
     internal class MetadataReferenceChecksumCollection : ChecksumCollection
     {
         public MetadataReferenceChecksumCollection(Checksum[] checksums) : this((object[])checksums) { }
-        public MetadataReferenceChecksumCollection(object[] checksums) : base(WellKnownSynchronizationKind.MetadataReferenceChecksumCollection, checksums) { }
+        public MetadataReferenceChecksumCollection(object[] checksums) : base(checksums) { }
```

Comment:
💭 For this case, the type seems desirable.

Reply:
how come?  afaict, nothing reads these.